### DISCUSSION TITLE
Form level resolver together with field level validation

### DISCUSSION THREAD

**Discussion by ludvigeriksson** [react-hook-form/react-hook-form#5274] [Q&A]
Is is possible to have both a form level resolver (to make cross field validations) as well as regular field validation rules? When I add a resolver to `useForm` my field validators no longer gets called.

**Comment by leapful** [comment]
@ludvigeriksson 

Could you please share more detail about the reason why you need both "resolver" and "field validator" to be triggered ? Usually, if "resolver" is used, there is no need to use "field validation" because "resolver" is a global validation which can validate all fields as a single source of truth.

**Reply by ludvigeriksson** [reply]
I don't want to have to revalidate the entire form every time. Since field validators are triggered onChange after the first submit, I don't want them all to run on every keystroke.

**Reply by bluebill1049** [reply]
you can achieve that behavior, however the schema library that you are using needs to support that. 

<img width="1021" alt="Screen Shot 2021-05-28 at 10 16 56 am" src="https://user-images.githubusercontent.com/10513364/119911993-d89ef980-bf9d-11eb-8cae-7233211b8747.png">

we do provide which field that is currently getts validated and plus the entire form value which supplied. at the moment, it's not an option to do so natively because not all schema library does support such behavior, as far as I know only vest do support such validation per field.
